src/plugins/intel_gna/src/gna_graph_tools.hpp

```diff
@@ -130,24 +130,30 @@ inline typename ExtractRawPtr<T>::raw_ptr_type raw_ptr(T obj) {
  * @brief gets pointer to previous layer
  * @param idx - index in previous layer connection - in other layers only zero idx will be used
  * @param layer - source layer
- * @param shouldSkip - skip kriteria
+ * @param shouldSkip - skip criteria
+ * @param noThrow - doesn't throw exception if previous layer missed
  */
 template <class Layer>
 inline InferenceEngine::CNNLayerPtr CNNNetPrevLayerSkipCertain(Layer layer,
                                                                int idx,
-                                                               const std::function<bool(CNNLayerPtr)>& shouldSkip) {
+                                                               const std::function<bool(CNNLayerPtr)>& shouldSkip,
+                                                               bool noThrow = false) {
```

Comment:
I think we're still not following the parameter naming convention.

Reply:
fixed